### PR TITLE
add volk/1.2.198.0

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **volk/1.2.198**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/volk/all/conandata.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.198.0":
+    url: "https://github.com/zeux/volk/archive/1.2.198.tar.gz"
+    sha256: "8f3869d94ccce265eaef064b1be0af2d8de925e512d3413c1cf02591b99597e2"
```
Comment (ericLemanissier) [inline recipes/volk/all/conandata.yml:4]: why the additional `.0` in the version number ?
also, [1.2.198](https://github.com/zeux/volk/releases/tag/1.2.198) tag is actually the commit 713ab8bfcd9d8e5fe78ccf85806c0b4e5c566f71, which is used for version 1.2.195 below. Why is that ?

Comment (AndreyMlashkin) [reply recipes/volk/all/conandata.yml:4]: volk is updated not for each minor version of vulkan-headers.
Recipe is implemented the way, it's version should match vulkan-headers version. There is only '1.2.198.0' version and no '1.2.198' version

**Comment by SpaceIm** [comment]
The title is misleading.

File: recipes/volk/all/conandata.yml (hunk 1 above)
Comment (madebr) [reply recipes/volk/all/conandata.yml:4]: I would drop the `.0`.
I [can't see the suffix](https://github.com/zeux/volk/releases/tag/1.2.190).

You say the volk version number must match the vulkan version.
How come the `.0` suffix was not needed before?

Comment (SpaceIm) [reply recipes/volk/all/conandata.yml:4]: You can add `vulkan-headers/1.2.198`, which is exactly the same commit than `vulkan-headers/1.2.198.0` already packaged.

Comment (AndreyMlashkin) [reply recipes/volk/all/conandata.yml:4]: If we have both vulkan-headers/1.2.198 and vulkan-headers/1.2.198.0, why can't we have two versions of volk then?

File: recipes/volk/all/conandata.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.198.0":
```
Comment (SSE4) [inline recipes/volk/all/conandata.yml:2]: should it have a trailing zero? other versions don't have

Comment (AndreyMlashkin) [reply recipes/volk/all/conandata.yml:2]: volk version should match vulkan-header version and there is no 1.2.198 yet, only one with a trailing zero.

However, you can approve this MR:
https://github.com/conan-io/conan-center-index/pull/8187#pullrequestreview-818214355
then I could remove .0

**Pull request closed without merging**